lib/Pod/To/HTML.pm

```diff
@@ -45,8 +45,9 @@ my @body;
 my @footnotes;
 my %crossrefs;
 
- sub Debug(Callable $)  { }         # Disable debug code
-#sub Debug(Callable $c) { $c() }    # Enable debug code
+#sub Debug(Callable $)  { }         # Disable debug code
+sub Debug(Callable $c) { $c() }    # Enable debug code
```

Comment:
I'm unfamiliar with this codebase, but this looks like the Debug mode is still enabled?